src/main/features/offer/views/declaration.njk

```diff
@@ -45,35 +45,47 @@
         }}
       </p>
 
-      <p>{{ t('If you both keep to this agreement:') }}</p>
-      <ul class="list list-bullet">
-        <li>{{ t('the original claim and any counterclaims will be closed') }}</li>
-        <li>{{ t('neither of you can make any further claims relating to this case') }}</li>
-      </ul>
+      <p>{{ t('This agreement replaces the claim. Neither of you can make any further claims relating to this case.') }}</p>
 
       <p>{{ t('This agreement is confidential until you’ve both signed it. After that it can be used in court, for example if the agreement is broken.') }}</p>
 
+      <p>{{ t('The original claim and any counterclaims will be kept on hold in case this agreement is broken.') }}</p>
+
+      <p>{{ t('You can view and download this agreement at any time from your money claims account.') }}</p>
+
       <h3 class="heading-medium">{{ t('If the agreement is broken') }}</h3>
 
       <p>{{ t('Either of you can apply to:') }}</p>
       <ul class="list list-bullet">
         <li>{{ t('request a judgment for any money still owed from this agreement') }}</li>
-        <li>{{ t('restart the original claim or counterclaim') }}</li>
+        <li>{{ t('restart the original claim or counterclaim for the full amount of the claim') }}</li>
       </ul>
 
       <p>
         {{
-          t('<a href="{{ n244formUri }}">Complete form N244</a> and send it to the <a href="{{ countyCourtBusinessCentreUri }}">County Court Business Centre</a>. You need to say on the form which of the 2 options you’re applying for.', {
+          t('You should complete <a href="{{ n244formUri }}">form N244</a> and send it to the <a href="{{ countyCourtBusinessCentreUri }}">County Court Business Centre</a>. You need to say on the form which of the 2 options you’re applying for.', {
             n244formUri: 'http://formfinder.hmctsformfinder.justice.gov.uk/n244-eng.pdf',
             countyCourtBusinessCentreUri: 'https://courttribunalfinder.service.gov.uk/courts/county-court-business-centre-ccbc'
           }) | safe
         }}
       </p>
 
-      <p>{{ t('The original claim and any counterclaims will be kept on hold in case this agreement is broken. They will be closed three months after the last payment date (if applicable) if we don’t hear from either of you.') }}</p>
-
       <h2 class="heading-medium">{{ t('Sign the agreement') }}</h2>
 
+      <p>{% set signAgreementText = 'Make sure this contract includes everything you have agreed with {{ name }} before signing.' %}</p>
+
+        {% if user.id === claim.defendantId  %}
```

Comment:
this would be simpler if the name was set in the var and then interpolated in the <p>
`<p>{{ t('Make sure... {{ name }}, { name: name }) }}</p>`

REC

Reply:
Done